packages/sycamore-core/src/render.rs

```diff
@@ -374,3 +370,219 @@ pub fn reconcile_fragments<G: GenericNode>(parent: &G, a: &mut [G], b: &[G]) {
         }
     }
 }
+
+fn remove_old_nodes<G: GenericNode>(
```

Comment:
It would be nice to have a doc-comment explaining what this function does, but it can be really short since it's not public.